email/blocker.go

```diff
@@ -176,11 +188,17 @@ func (b *Blocker) blockReport(report database.AbuseReport) []string {
 			}
 
 			// execute the request
-			b.staticLogger.Debugf("blocking %v", skylink)
+			b.staticLogger.Debugf("blocking %v...%v", skylink[:4], skylink[len(skylink)-4:])
```

Comment:
(NOTE: this is temporary, once the blocker only deals with hashes, the abuse scanner will do so too and this will get removed, figured in the mean time it makes sense to at least obfuscate it from the logs already)